.mergify.yml

```diff
@@ -5,11 +5,23 @@ queue_rules:
       - base=master
       # Require integration tests before merging only
       - or:
+          - and:
+              - '-check-failure=deployment-test'
+              - or:
+                  - '#commits-behind>0'
+                  - queue-position<0
+          - label=bypass:integration
           - check-success=deployment-test
           - check-neutral=deployment-test
           - check-skipped=deployment-test
-          - label=bypass:integration
```

Comment:
What is the consequence of removing this line?

Reply:
Not removed, moved up